src/main/java/controller/GameController.java

```diff
@@ -1,53 +1,59 @@
 package controller;
 
 import domain.Car;
+import domain.MovingStragey;
 import domain.Winner;
+import util.CarNameUtil;
 import util.RandomNumUtil;
-import util.StringUtil;
 import view.InputView;
 import view.ResultView;
 
 import java.util.ArrayList;
-import java.util.Comparator;
 import java.util.List;
 
 public class GameController {
 
-    List<Car> participant;
-    private int gameCount;
+    private List<Car> participants;
     private InputView inputView;
     private Winner winners;
 
     public GameController() {
-        participant = new ArrayList<>();
+        participants = new ArrayList<>();
         inputView = new InputView();
         winners = new Winner();
     }
 
-    public void init() throws IllegalArgumentException {
-        String[] carNames = StringUtil.participants(inputView.participantInputView());
+    public void playRacingGame() throws IllegalArgumentException {
+        String[] carNames = CarNameUtil.participantsSplit(inputView.participantInputView());
 
         for (int count = 0; count < carNames.length; count++) {
-            participant.add(new Car(carNames[count]));
+            participants.add(new Car(carNames[count]));
         }
-        gameCount = inputView.playInputView();
+        startGame(inputView.playInputView());
+        endGameResult();
+    }
+
+    public void startGame(int gameCount) {
         for (int count = 0; count < gameCount; count++) {
-            playGame();
-            ResultView.racingView(participant);
+            racing();
+            ResultView.racingView(participants);
         }
-        winners.createWinnersData(participant, readWinnerCarDistance());
-        ResultView.winnersView(winners.readWinners());
     }
 
-    public void playGame() {
-        for(int count = 0; count < participant.size(); count++) {
-            participant.get(count).carMoveBehavior(RandomNumUtil.makeRandomNumber());
+    private void racing() {
+        for (Car entryCar : participants) {
+            entryCar.move(new MovingStragey() {
+                @Override
+                public int carMove() {
+                    return super.carMove();
+                }
+            });
```

Comment:
```suggestion
            entryCar.move(new MovingStragey());
```

와 같이 작성할 수 있을 것 같은데 Override하도록 작성해주신 이유가 있을까요!?